.github/workflows/ui-tests-periodic.yaml

```diff
@@ -0,0 +1,47 @@
+name: UI Tests Periodic
+
+on:
+  schedule:
+    - cron: '0 5 * * 1' # run every Monday at 5:00 AM
+
+permissions:
```

Comment:
Why do we need this? I think read should be provided per default for the GitHub token in Actions.